src/api/java/baritone/api/Settings.java

```diff
@@ -840,16 +840,10 @@ public final class Settings {
      */
     public final Setting<Integer> allowOnlyExposedOresDistance = new Setting<>(1);
 
-
-    /**
-     * Only mine blocks out of the selected area. Can't be combined with {@link #onlyMineInSelection}
-     */
-    public final Setting<Boolean> onlyMineOutOfSelection = new Setting<>(false);
-
     /**
-     * Only mine blocks in the selected area. Can't be combined with {@link #onlyMineOutOfSelection}
+     * Only mine blocks in the selected area.
      */
-    public final Setting<Boolean> onlyMineInSelection = new Setting<>(false);
+    public final Setting<Boolean> mineOnlyOutsideOfSelection = new Setting<>(false);
```

Comment:
I think you made an oopsie here.

Reply:
Yep, already fixed it, noticed it whilst testing making this setting also work for farm